std_msgs/Byte number
std_msgs/Bool sex
std_msgs/Byte age

---
